src/material/button/_button-theme-private.scss

```diff
@@ -19,6 +19,7 @@
     opacity: map.get($opacities, hover);
   }
 
+  &:focus,
   &.cdk-program-focused,
   &.cdk-keyboard-focused {
```

Comment:
I think the point of `cdk-program-focused` and `cdk-keyboard-focused` below was not to use `:focus` which also shows up for things like mouse clicks.

Reply:
What triggers these classes getting appended? From what I can see, the regular/default button doesn't get these so the focus state never changes the background color

Reply:
They get added if `FocusMonitor.monitor` is called on the button and the user tabs into it or `.focus()` is called on it directly. Maybe the `FocusMonitor` call got lost somewhere in the refactors?

Reply:
Is the focus state not meant to show up for mouse clicks?

Reply:
Yeah that was one of the reasons we introduced the FocusMonitor.